test/screenshot/lib/image-cropper.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const jimp = require('jimp');
+
+class ImageCropper {
```

Comment:
I'd be tempted to amalgamate this with the `ImageDiffer` just because the crop size will absolutely affect the diff if the crop dimensions change.

Reply:
I considered that, but I wanted to upload the cropped image to GCS.

Mainly because A) it makes the images easier to review, and B) it makes it easier to tell if the cropper isn't working properly.

Does that make sense? Am I missing anything?

Reply:
I get it now. The `ImageCropper` is used when the image is downloaded from CBT. It makes sense. Thanks for explaining!